test/extensions/INTEL/SPV_INTEL_function_pointers/bitcast.ll

```diff
@@ -37,8 +39,8 @@ define dso_local spir_func signext i8 @foo(i8 signext %0) #0 {
 
 ; Function Attrs: noinline nounwind optnone
 define dso_local spir_func void @bar() #0 {
-  %1 = alloca i32 (i32)*, align 4
-  store i32 (i32)* bitcast (i8 (i8)* @foo to i32 (i32)*), i32 (i32)** %1, align 4
+  %1 = bitcast i8 (i8)* @foo to i32 (i32)*
+  %2 = call i32 %1(i32 0)
```

Comment:
@AlexeySachkov may I ask you to take a look if you are OK with the test change

Reply:
It looks like this IR doesn't match OpenCL C snippet in the comments above. I'm fine with the IR change, but we need to align OpenCL C snippet then to add call there.